src/utils/constants.ts

```diff
@@ -581,13 +581,58 @@ export const TroubleshootingFilters = (intl: IntlShape) => {
   const data = {
     name: intl.formatMessage({ id: 'troubleshooting_filter_module.title' }),
     options: [
-      { id: 'Catalog', name: 'Catalog' },
-      { id: 'Order', name: 'Order' },
-      { id: 'Shipping', name: 'Shipping' },
-      { id: 'VTEX Admin', name: 'VTEX Admin' },
-      { id: 'Prices', name: 'Prices' },
-      { id: 'Message Center', name: 'Message Center' },
-      { id: 'Trade Policy', name: 'Trade Policy' },
+      {
```

Comment:
I am going to try and generate a list with the script that rearranges the tags. But for now we can use a manual approximation, even if we don't check every article.